dashboard/app/views/home/_homepage.html.haml

```diff
@@ -12,7 +12,7 @@
 - if current_user
   - homepage_data[:providers] = current_user.providers
   - if current_user.teacher?
-    - homepage_data[:hocLaunch] = DCDO.get('hoc_mode', CDO.default_hoc_mode) == 'actual-hoc' ? 'actual-hoc' : DCDO.get('hoc_launch', CDO.default_hoc_launch)
+    - homepage_data[:hocLaunch] = DCDO.get('hoc_launch', CDO.default_hoc_launch)
```

Comment:
i think we can totally remove `hocLaunch` since our component no longer uses it, right?

Reply:
(this can be follow-up work since you're trying to get this out the door)

Reply:
Yeah I also had an inkling that this could be useful plumbing to leave intact for next year.